Greenshot/Languages/language-en-US.xml

```diff
@@ -9,6 +9,7 @@
 Greenshot comes with ABSOLUTELY NO WARRANTY. This is free software, and you are welcome to redistribute it under certain conditions.
 Details about the GNU General Public License:</resource>
 		<resource name="about_title">About Greenshot</resource>
+		<resource name="about_translation">Info about translation</resource>

```

Comment:
```suggestion -1+1
		<!-- uncomment and modify example text to add credits or other translation-related info (displayed in about dialog)
		<resource name="about_translation">English translation by Jane Doe</resource>
		-->
```

I think we had a comment in here a while ago explaining how to add translator credits, I don't know when and why it got lost.

I prefer this variant, as it does not show up in the english about dialog :smile:  what do you think?

Reply:
@Lakritzator 

Your propsoal could be an idea,.

Please take care that I I used for english language the same identifier of translated langusge file because otherwise if you make a compare you will find one orphan string (in translated language).

For that I propose my solution that use the same identifier of translated language file.

I just explain the reason of my choice.

Reply:
Thanks for getting back. I just changed my proposal so that the comment is now included within the "about_translation" entry, so the XML node for that `name` exists (but is empty). Having an acutal text node in there would make it necessary to have language-specific logic in our about dialog.

I know, translating Greenshot is currently a bit cumbersome. Some time in the future we'll replace this file-based pull request workflow with some platform approach.

Thanks for your contribution.